app/controllers/tasks_controller.rb

```diff
@@ -5,22 +5,22 @@ class TasksController < ApplicationController
   def create
     @task.set_status
     if @task.save
-      redirect_to project_tasks_url(@task.project), notice: "Returing from the create"
+      redirect_to project_tasks_url(@task.project), notice: "Task created successfully!"
     else
-      render new_project_task_path(@task.project), notice: "Cannnot enter data due to constraints"
+      redirect_to new_project_task_path(@task.project), notice: "Task cannot be created!"
```

Comment:
what if  @task.save because of "project_id" then code will be break in else because @task.project will be nil or something not desirable.